pkg/roachprod/vm/gce/gcloud.go

```diff
@@ -1033,6 +1035,8 @@ func (o *ProviderOpts) ConfigureCreateFlags(flags *pflag.FlagSet) {
 	_ = flags.MarkDeprecated("machine-type", "use "+ProviderName+"-machine-type instead")
 	flags.StringSliceVar(&o.Zones, "zones", nil, "DEPRECATED")
 	_ = flags.MarkDeprecated("zones", "use "+ProviderName+"-zones instead")
+	flags.StringSliceVar(&o.ManagedSpotZones, ProviderName+"-managed-spot-zones", nil,
+		"use this to specify in which zones managed instance groups will use spot instances")
```

Comment:
Nit: IMO the `use this` part doesn't add much. "subset of zones in managed instance groups that will use spot instances" is more straight to the point IMO.

Reply:
That's more concise, thanks! updated.